### PR TITLE
Feat/analytics dropdown

### PR DESCRIPTION
**Pull request by matthew-garrett**
- Adds markets dropdown
- Adds network to markets route
- Adds accounts menu for all accounts and vault accounts

### CODE REVIEW

File: gcp/analytics/src/utils/notional-utils.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export const capitalizeFirstLetter = (string) => {
```
Comment (jeffywu) [inline gcp/analytics/src/utils/notional-utils.ts]: Isn't there a CSS property for this?

File: packages/common/icons/src/utils.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+export enum Network {
```
Comment (jeffywu) [inline packages/common/icons/src/utils.tsx]: I'm not sure this is a good idea...we should try to actually fix the import issue.

File: gcp/analytics/src/layout/DashboardLayout/Header/HeaderContent/index.tsx [hunk 3]
```diff
@@ -48,7 +49,25 @@ export default function Header() {
     }
   }, [pathname, setPathname]);
 
-  const availableTokens = ['ETH', 'DAI', 'USDC', 'GHO'];
+  const mainnetTokens: string[] = ['USDC', 'rETH', 'USDT', 'ETH', 'wstETH', 'DAI', 'sDAI', 'ARB', 'GHO', 'wBTC', 'FRAX'];
```
Comment (jeffywu) [inline gcp/analytics/src/layout/DashboardLayout/Header/HeaderContent/index.tsx]: Can we move this off into a hook? We will need to fetch this list from an API.

File: gcp/analytics/src/layout/DashboardLayout/Header/HeaderContent/index.tsx [hunk 4]
```diff
@@ -225,28 +280,79 @@ export default function Header() {
...
-          }}
-        >
-          {availableTokens.map((token, i) => (
-            <NextLink href={`/markets/${token}/overview`} passHref legacyBehavior key={i}>
-              <Link className="header-link" color="black" underline="none" sx={{ cursor: 'pointer' }}>
-                {token}
-              </Link>
-            </NextLink>
-          ))}
+        <Box sx={{ width: '767px', height: '100%', display: 'flex', justifyContent: 'space-between' }}>
```
Comment (jeffywu) [inline gcp/analytics/src/layout/DashboardLayout/Header/HeaderContent/index.tsx]: We're using a lot of hardcoded `px` values in here, is that the convention for this project or do they have some sort of spacing method they use from the theme?

File: packages/common/icons/src/utils.tsx (hunk 2 above)
Comment (matthew-garrett) [reply packages/common/icons/src/utils.tsx]: It was being a pain in the ass and didn't seem worth it to me for one function and one enum

**Pull request merged**